Cargo.toml

```diff
@@ -45,6 +45,9 @@ matrixmultiply = { version = "0.3.2", default-features = false, features=["cgemm
 serde = { version = "1.0", optional = true, default-features = false, features = ["alloc"] }
 rawpointer = { version = "0.2" }
 
+portable-atomic = { git = "https://github.com/taiki-e/portable-atomic.git", rev = "4e66bd3", optional = true }
+portable-atomic-util = { git = "https://github.com/taiki-e/portable-atomic.git", rev = "4e66bd3", features = ["alloc"], optional = true}
```

Comment:
crates with git dependencies can't be published to crates.io, so we can't merge this PR for that reason.

Reply:
@bluss This should be fixed now! Thank you for the feedback!